templates/main.php

```diff
@@ -51,35 +49,38 @@ class="collection"
+                        <button></button>
+                    </span>
+                    <div class="app-navigation-entry-menu" ng-show="calendar.writable">
+                        <ul>
+                            <li>
+                                <a ng-click="startEdit(calendar)">
+                                    <span class="icon-rename svg"></span>
+                                    <span><?php p($l->t('Edit')); ?></span>
+                                </a>
+                            </li>
+                            <li>
+                                <a ng-click="showCalDAVUrl(calendar)">
+                                    <span class="icon-public svg"></span>
+                                    <span><?php p($l->t('Link')); ?></span>
+                                </a>
+                            </li>
+                            <li>
+                                <a href="{{calendar.exportUrl}}" download="{{calendar.uri}}.ics">
+                                    <span class="icon-download svg"></span>
+                                    <span><?php p($l->t('Download')); ?></span>
+                                </a>
+                            </li>
+                            <li confirmation="delete(calendar)" confirmation-message="deleteMessage(calendar)">
+                            </li>
+                        </ul>
+                    </div>
                 </div>
-                <div class="app-navigation-entry-menu" ng-show="calendar.writable">
-                    <ul>
-                        <li>
-                            <span ng-click="startEdit(calendar)">
-                                <img class="icon-rename svg" src="<?php p(image_path('core', 'actions/rename.svg'))?>"/>
-                                <span><?php p($l->t('Edit')); ?></span>
-                            </span>
-                        </li>
-                        <li>
-                            <span ng-click="showCalDAVUrl(calendar)">
-                                <img class="icon-public svg" src="<?php p(image_path('core', 'actions/public.svg'))?>"/>
-                                <span><?php p($l->t('Link')); ?></span>
-                            </span>
-                        </li>
-                        <li>
-                            <a href="{{calendar.exportUrl}}" download="{{calendar.uri}}.ics">
-                                <img class="icon-download svg" src="<?php p(image_path('core', 'actions/download.svg'))?>"/>
-                                <span><?php p($l->t('Download')); ?></span>
-                            </a>
-                        </li>
-                        <li confirmation="delete(calendar)" confirmation-message="deleteMessage(calendar)">
-                        </li>
-                    </ul>
+                <div class="app-navigation-entry-utils-counter">
```

Comment:
Sorry, I thought you said that you will align it, not that you wanted to :)
Well, I don't think this is against the guidelines, but if you hide the counter properly, the menu should align automatically on the right :)